quill-async/src/main/scala/io/getquill/context/async/AsyncContext.scala

```diff
@@ -56,7 +56,7 @@ abstract class AsyncContext[D <: SqlIdiom, N <: NamingStrategy, C <: Connection]
       f(TransactionalExecutionContext(ec, c))
     }
 
-  def executeQuery[T](sql: String, prepare: List[Any] => (List[Any], List[Any]) = row => (Nil, row), extractor: RowData => T = identity[RowData] _)(implicit ec: ExecutionContext): Future[List[T]] = {
+  def executeQuery[T](sql: String, prepare: Prepare = identityPrepare, extractor: Extractor[T] = identityExtractor)(implicit ec: ExecutionContext): Future[List[T]] = {
```

Comment:
@mxl  BTW, idea (2017.1.2 c) complains on `identityExtractor`, making it as `identityExtractor _` resolves that, do you have something like this?